@navikt/core/react/src/stepper/Stepper.tsx

```diff
@@ -26,6 +26,7 @@ export interface StepperProps extends React.HTMLAttributes<HTMLOListElement> {
    */
   onStepChange?: (step: number) => void;
   /**
+   * @deprecated Use `interactive` prop on `<Stepper.Step />` instead for individual steps. For completely static steppers, use `Process` component instead.
    * Makes stepper non-interactive if false.
```

Comment:
The second line becomes a part of the "deprecated"-block now. Was that intended?

<img width="930" height="177" alt="Image" src="https://github.com/user-attachments/assets/a1df40e0-226f-48b4-85bf-f47d86a60c6b" />


Either add a blank line between, or switch the order.

```suggestion
   * Makes stepper non-interactive if false.
   * @deprecated Use `interactive` prop on `<Stepper.Step />` instead for individual steps. For completely static steppers, use `Process` component instead.
```